gradle.properties

```diff
@@ -15,7 +15,7 @@
 #
 projectGroup=com.ritense.valtimo
 projectName=valtimo-backend-libraries
-kotlinVersion=1.6.0
+kotlinVersion=1.6.10
```

Comment:
Don't forget to include this in the Valtimo Docs - Version compatibility matrix for the next release (candidate)